seqr/utils/elasticsearch/constants.py

```diff
@@ -108,6 +108,8 @@
     'AN': {},
     'Hom': {},
     'Hemi': {},
+    'Het': {},
```

Comment:
you've added this to the parsed data but it doesn't look like we are displaying it anywhere?

Reply:
Ooh... I thought it could be display automatically by the frontend.
Updated now, and it is displayed as below:
![image](https://user-images.githubusercontent.com/33550059/122413457-62fecb80-cf54-11eb-9f42-519bc5c72d7d.png)